### PR TITLE
Create WordsMatter.md

### LINKED ISSUE #1865: Additional inclusive language resource
Author: @rinkug 
EB member: @markcmiller86 

https://www.acm.org/diversity-inclusion/words-matter

This is a list of problematic terms and suggested replacements.  The list apparently originated with the NSF XSEDE project and they handed it to the ACM DEI group who (according to the website) plan to continue curating it.

### PR DESCRIPTION
**Pull request by rinkug**
# Description
Author: @rinkug 
EB Member: @markcmiller86 
Resolves #1865 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [x] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

### CODE REVIEW

**Comment by rinkug** [comment]
@markcmiller86 : Can you help review this? Its right up your alley and should be quick for you, I am guessing.

File: CuratedContent/WordsMatter.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Resource information | Details
+:--- | :---
+Resource title | Words Matter
+Authors | ACM Diversity and Inclusion Council
+Website | [ACM - Words Matter](https://www.acm.org/diversity-inclusion/words-matter)
+
+The ACM Diversity and Inclusion Council emphasizes the significant impact of language in the computing profession through their article [Words Matter](https://www.acm.org/diversity-inclusion/words-matter). 
+This effort aims to replace offensive or exclusionary terminology with alternatives that are clearer and more inclusive. 
+Examples of exclusionary words include "master/slave," "blacklist/whitelist," and "blind review". 
+The Council collaborates with major organizations like IBM, Apple, and Github in this endeavor.
```
Comment (markcmiller86) [inline CuratedContent/WordsMatter.md:19]:
```suggestion
The Council collaborates with major organizations like IBM, Apple, and Github. 
```

File: CuratedContent/WordsMatter.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+:--- | :---
+Resource title | Words Matter
+Authors | ACM Diversity and Inclusion Council
+Website | [ACM - Words Matter](https://www.acm.org/diversity-inclusion/words-matter)
+
+The ACM Diversity and Inclusion Council emphasizes the significant impact of language in the computing profession through their article [Words Matter](https://www.acm.org/diversity-inclusion/words-matter). 
+This effort aims to replace offensive or exclusionary terminology with alternatives that are clearer and more inclusive. 
+Examples of exclusionary words include "master/slave," "blacklist/whitelist," and "blind review". 
+The Council collaborates with major organizations like IBM, Apple, and Github in this endeavor. 
+They recommend using terms like "primary/secondary," "allowlist/blocklist," and "anonymous review" instead.
```
Comment (markcmiller86) [inline CuratedContent/WordsMatter.md:20]:
```suggestion
The article recommends alternatives such as "primary/secondary," "allowlist/blocklist," and "anonymous review", repsectively. 
```

**Review by markcmiller86 — APPROVED** [review]
Looks good. I proposed a couple of changes. Feel free to ignore them though too. 

Also, FWIW, I really try NOT to emphasize the use of "offensive" as a criterion for whether some terminology is inclusive or not. That is because "offensive"-ness is highly subjective...nobody will agree on what constitutes offense.

**Pull request merged**